src/lib/config.js

```diff
@@ -18,6 +18,7 @@ const DEFAULT_SLIPPAGE = 0.001 // 0.1%
 const DEFAULT_ROUTE_BROADCAST_INTERVAL = 30 * 1000 // milliseconds
 const DEFAULT_ROUTE_CLEANUP_INTERVAL = 1000 // milliseconds
 const DEFAULT_ROUTE_EXPIRY = 45 * 1000 // milliseconds
+const DEFAULT_QUOTE_EXPIRY = 20 * 1000 // milliseconds
```

Comment:
I think default should be 45 because routes are broadcast only ever 30 seconds. So setting the quote expiry to 20 seconds would lead to remote quoting even where that's not desired